packages/app/src/runner/event-manager.ts

```diff
@@ -432,9 +432,9 @@ export class EventManager {
     }
 
     Cypress = this.Cypress = this.$CypressDriver.create(config)
+    this.localBus.emit('cypress:created', Cypress)
```

Comment:
can we add a test that this gets emitted correctly?

Reply:
I thought about it but we don't have any unit tests and it seemed heavy handed to add a cy-in-cy test for it.

Reply:
Maybe I can update an existing test for it. I'll take a look.

Reply:
If it's too much effort I wouldn't worry about it

Reply:
added test [39e67a1](https://github.com/cypress-io/cypress/pull/31399/commits/39e67a1b605b9159ffbad0f54197df408381e2e0)